client/src/components/Item.jsx

```diff
@@ -1,13 +1,87 @@
 import React from 'react';
+import styled from 'styled-components';
+import stars from './Assets/stars.jpeg';
+
+const Card = styled.li`
+  list-style-type: none;
+  padding: 5px;
+  border: 1px solid red;
+  margin: 5px;
+  width: 30%;
+  float: left;
+`;
+
+const GenCaption = styled.h1`
```

Comment:
What is your reasoning behind using !important in some of your styles? I don't think it's an issue, but I'm curious as to what your thought process was.

Reply:
Sure.` !important ` overrides all other css rules cascaded from elsewhere. 

[https://stackoverflow.com/questions/9245353/what-does-important-mean-in-css]